cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx

```diff
@@ -403,6 +413,15 @@ export class ToolsControlComponent extends React.PureComponent<Props, State> {
         });
     };
 
+    private onChangeBlockState = (event:string):void => {
+        const { activeControl, onSwitchBlockMode } = this.props;
+        if ([ActiveControl.AI_TOOLS].includes(activeControl) && event === 'keydown') {
```

Comment:
What is wrong with ``isActivated`` prop which is ``activeControl === ActiveControl.AI_TOOLS``?

Reply:
Done.